source/assets/ur5e/actuator_models/actuator_model_v1.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "UR5E",
+    "version": "0.0.1",
+    "description": "UR5E actuator model trained on dataset (v1 https://drive.google.com/file/d/1G1V-y0hHAjVN6Uy34iNlVfRDm1VV5dij/view?usp=drive_link)",
```

Comment:
Is this public?

Reply:
For this version yes, but for later actuator model training we can keep it proprietary